opencti-platform/opencti-graphql/src/manager/ingestionManager.ts

```diff
@@ -50,6 +52,43 @@ const asArray = (data: unknown) => {
   }
   return [];
 };
+
+const updateBuiltInConnectorInfo = async (context: AuthContext, id: string, state: object) => {
+  // Patch the related connector
+  const connectorId = uuidv5(id, OPENCTI_NAMESPACE);
+  const csvNow = utcDate();
+  const connectorPatch: any = {
+    updated_at: csvNow.toISOString(),
+    connector_info: {
+      last_run_datetime: csvNow.toISOString(),
+      next_run_datetime: csvNow.add(SCHEDULE_TIME, 'milliseconds').toISOString(),
+      run_and_terminate: false,
+      buffering: false,
+      queue_threshold: 0,
+      queue_messages_size: 0
+    },
+    connector_state: JSON.stringify(state)
+  };
+  await patchAttribute(context, SYSTEM_USER, connectorId, ENTITY_TYPE_CONNECTOR, connectorPatch);
+};
+
+const pushBundleToConnectorQueue = async (context: AuthContext, ingestion: BasicStoreEntityIngestionTaxii
+| BasicStoreEntityIngestionRss | BasicStoreEntityIngestionCsv, bundle: StixBundle) => {
+  // Push the bundle to absorption queue
+  const connector = { internal_id: uuidv5(ingestion.id, OPENCTI_NAMESPACE), connector_type: ConnectorType.ExternalImport };
+  const workName = `run @ ${now()}`;
+  const work: any = await createWork(context, SYSTEM_USER, connector, workName, connector.internal_id, { receivedTime: now() });
```

Comment:
Maybe we could create and use an INGESTION_MANAGER_USER ?

Reply:
I dont think its needed as we dont track who create the work.